package.json

```
@@ -2,26 +2,48 @@
   "name": "web-template",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
   "scripts": {
-    "dev": "next dev",
     "build": "next build",
+    "dev": "next dev",
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "lint": "next lint",
```

trzymalbym sie jedenj konwencji czyli jak mamy format i format:check to zeby lint fixowal a lint:check sprawdzal, tylko wtedy update github actions, albo w druga strone, lint i lint:fix i format i format:fix. jak wolisz ale jednolicie 

```suggestion
    "lint": "next lint --fix",
    "lint:check": "next lint --max-warnings=0",
```